src/main/java/frc/robot/RobotContainer.java

```diff
@@ -26,18 +29,53 @@ public class RobotContainer {
 
     private final Telemetry logger = new Telemetry(MaxSpeed);
 
+
     private final CommandXboxController driverCont = new CommandXboxController(0);
     private final CommandXboxController operatorCont = new CommandXboxController(1);
 
-    public final CommandSwerveDrivetrain driveTrain;
+    public CommandSwerveDrivetrain driveTrain;
+    private Vision vision;
+
+    private ShuffleboardTab diagnosticTab;
 
-    private final Vision vision = new Vision(new VisionIOLimelight("limelight", 0.0, 0.0));
 
     public RobotContainer() {
-      driveTrain = OldCompBot.createDrivetrain();
+        switch (Constants.getRobotType()) {
+            case WOODBOT:
+                //woodbot stuff
+                vision = new Vision(new VisionIOLimelight(
+                    Constants.VisionConstants.WOODBOT_LIMELIGHT_NAME, 
+                    Constants.VisionConstants.WOODBOT_YAW_FUDGE_FACTOR,
+                    Constants.VisionConstants.WOODBOT_PITCH_FUDGE_FACTOR));
+                driveTrain = OldCompBot.createDrivetrain();
+                break;
+            case OLD_COMP_BOT:
+                //ocb stuff
+                vision = new Vision(new VisionIOLimelight(
+                    Constants.VisionConstants.OCB_LIMELIGHT_NAME,
+                    Constants.VisionConstants.OCB_YAW_FUDGE_FACTOR,
+                    Constants.VisionConstants.OCB_PITCH_FUDGE_FACTOR));
+                driveTrain = OldCompBot.createDrivetrain();
+            case PRACTICE:
+                //practice bot stuff
+                break;
+            case COMPETITION:
```

Comment:
Let's have some extra security and set it up like this so the default case is also setting it to the comp bot like this
`case COMPETITION:`
`default:`
`  //competition bot stuff`
`  break;`